rotkehlchen/chain/substrate/typing.py

```diff
@@ -13,14 +13,22 @@
 class KusamaNodeName(Enum):
     """Public nodes for Kusama.
     """
-    PARITY = 0
+    OWN = 0
+    PARITY = 1
 
     def __str__(self) -> str:
+        if self == KusamaNodeName.OWN:
+            return 'own node'
         if self == KusamaNodeName.PARITY:
             return 'parity'
         raise AssertionError(f'Unexpected KusamaNodeName: {self}')
 
     def endpoint(self) -> str:
+        if self == KusamaNodeName.OWN:
```

Comment:
This is not bad, so no need to change it.

An alternative would be to pass the rpc endpoint as an optional argument here, so if it's passed (which you would do for own node) then you use the given url. This is what we do with ethereum.

Reply:
I saw it but also realised that `endpoint` in NodeName is not optional but required. I thought that if I have it in the instance why passing it around. Anyway, these are just nuances :)